private/pkg/app/appproto/appprotoexec/appprotoexec.go

```diff
@@ -148,7 +148,7 @@ func NewHandler(
 		}
 		return newProtocProxyHandler(logger, storageosProvider, runner, protocPath, pluginName), nil
 	}
-	return nil, fmt.Errorf("could not find protoc plugin for name %s", pluginName)
+	return nil, fmt.Errorf("could not find protoc plugin for name %s; Fix by installing protoc-gen-%s", pluginName, pluginName)
```

Comment:
It is installed through Go so it would likely be in PATH, I agree though that it could be worded slightly differently

All in all, I submitted the PR to possibly save someone a headache as the error traceback is not explicit and I had to go in the code to understand the issue

Will patch, thanks for the remark!